.pre-commit-config.yaml

```
@@ -1,10 +1,11 @@
 repos:
--   repo: https://github.com/ambv/black
-    rev: stable
-    hooks:
-    - id: black
-      language_version: python3.7
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
-    hooks:
-    - id: flake8
+  -   repo: https://github.com/python/black
+      rev: stable
```

I've been running into `black` version mismatches between local and CI dev environments here and in other projects. What do you think about pinning to a specific release of `black`?

```suggestion
      rev: 20.8b1 
```

---

I think we should use this pre-commit stuff in ci instead as in https://github.com/dask/dask/pull/6645